ocaml/xapi/xapi_crashdump.ml

```diff
@@ -11,22 +11,15 @@
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
  * GNU Lesser General Public License for more details.
  *)
-exception Not_implemented
-
-let nothrow f () = try f() with _ -> ()
-
-let create ~__context ~vM ~vDI =
```

Comment:
They can still find it in the git history though, and @jonludlam said that this API class isn't really used anyway.